apps/src/code-studio/ConfirmEmailModal.jsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import locale from '@cdo/locale';
+import color from '@cdo/apps/util/color';
+
+const styles = {
+  title: {
+    position: 'absolute',
+    left: 30,
+    top: 10,
+    color: color.dark_charcoal,
+    margin: '15px 0',
+  },
+  content: {
+    position: 'absolute',
+    left: 30,
+    top: 50,
+    right: 30,
+    bottom: 70
+  },
+  contentText: {
+    fontSize: 15,
+    lineHeight: '22px'
+  },
+  highlightRow: {
+    backgroundColor: color.default_blue,
+    color: color.white,
+  },
+  footer: {
+    position: 'absolute',
+    bottom: 25,
+    right: 30,
+    left: 30,
+  },
+  buttonPrimary: {
```

Comment:
Yeah — looking more carefully, it looks like this is already duplicated a few places.  Maybe worth cleaning up in a future PR, but shouldn't block this PR!